apps/code-examples/src/app/code-examples/ag-grid/data-entry-grid/basic/data-entry-grid-docs-demo-edit-modal.component.html

```diff
@@ -1,6 +1,6 @@
 <sky-modal>
   <sky-modal-header> Edit employee information </sky-modal-header>
-  <sky-modal-content>
+  <sky-modal-content (keyup.escape)="onPressEscape($event)">
```

Comment:
Can this behavior be added to our `sky-ag-grid-wrapper` component? It could stop propagation if the form is in edit mode, or let it bubble-up otherwise.

Reply:
I was able to implement this in `sky-ag-grid-wrapper` but the "edit mode" calculation did not work within this event handler. The user would need to tab outside of the grid to use escape.